modules/post/windows/gather/credentials/pulse_secure.rb

```diff
@@ -0,0 +1,359 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  include Msf::Post::Windows::Registry
+  include Msf::Post::Windows::UserProfiles
+  include Msf::Post::Windows::Priv
+  include Msf::Auxiliary::Report
+  prepend Msf::Exploit::Remote::AutoCheck
```

Comment:
```suggestion
```

Reply:
Suggested fix given that AutoCheck isn't doing what I thought it was doing, again sorry for the confusion.

Reply:
Done in dbd3f2e1bf